File: 01_gpio_digital/led_blink.py
import RPi.GPIO as GPIO
import time

PIN_LED = 4
GPIO.setmode(GPIO.BCM)
GPIO.setup(PIN_LED, GPIO.OUT)

for i in range(10):
    GPIO.output(PIN_LED, GPIO.HIGH)
    print("Led on")
    time.sleep(1)
    GPIO.output(PIN_LED, GPIO.LOW)
    print("Led off")
    time.sleep(1)

GPIO.cleanup()

File: 01_gpio_digital/led_blink.c
#include <wiringPi.h>
#define LED 4

int main (void)
{
  //wiringPiSetup () ;
  wiringPiSetupGpio() ;
  pinMode (LED, OUTPUT) ;
  for (int i=0; i<5; i++)
  {
    digitalWrite (LED, HIGH) ; delay(1000);
    digitalWrite (LED,  LOW) ; delay(1000);
  }
  return 0 ;
}